homeassistant/components/aws/notify.py

```diff
@@ -36,48 +36,36 @@
 
 SUPPORTED_SERVICES = ["lambda", "sns", "sqs"]
 
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We should move this to the component and probably not extend the notify platform schema but make one from scratch.

Reply:
Not done yet, will do it later today

Reply:
Done